<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>beforeTurnOffHintWidget</class>
 <widget class="QDialog" name="beforeTurnOffHintWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>140</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>424</width>
    <height>140</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>424</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bluetooth</string>
  </property>
  <property name="windowIcon">
   <iconset theme="bluetooth">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>79</y>
     <width>96</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Turn Off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>79</y>
     <width>96</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>19</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>16</y>
     <width>341</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>Using Bluetooth mouse or keyboard, Do you want to turn off bluetooth?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
